pallets/erc20-xcm-bridge/src/xcm_exec_filter.rs

```diff
@@ -76,6 +76,8 @@ where
 						fun: WildNonFungible,
 						..
 					}) => {}
+					MultiAssetFilter::Wild(AllCounted(_amount)) => {}
```

Comment:
TODO: review with @librelois

Reply:
Anyway I plan to remove this filter. Either there will be no need for it anymore because we'll customize directly the XCM executor, or I'll replace it by a barrier so that it works everywhere and not only from local execution. I'm preparing a PR on polkadot upstream to make the XCM executor customizable to apply the 1st solution.
In the meantime you can remove this filter in this PR.

Reply:
done